### ISSUE TITLE
[ENHANCEMENT] Add naive and top-k methods for conformal classification

### ISSUE THREAD

**Issue by vincentblot28** [scikit-learn-contrib/MAPIE#100]
The naive method principle is to take all the classes with the higher softmax output such that their sum is almost equal to 1-alpha (the confidence level). It would able the user to compare conformal predictions to a simpler method. 

For the top-k method, the goal is to watch the distribution of k (k being the rank of the label in the sorted softmax output). Then take the 1-alpha quantile of this distribution, giving k*, which will be the size of the prediction set (always takings values from the higher to the lower one)

**Comment by vincentblot28** [comment]
Those two methods have been included in the pull request #111
